winter/pagination/page_position_argument_resolver.py

```diff
@@ -24,11 +35,35 @@ def resolve_argument(self, argument: ComponentMethodArgument, http_request: DRFR
 
         limit = self._parse_int_param(raw_limit, 'limit', min_value=1)
         offset = self._parse_int_param(raw_offset, 'offset', min_value=0)
+
+        self._try_redirect_to_default_limit(http_request, limit)
+
+        if limit is None:
+            limit = self.default_limit
+
+        if limit is not None and self.maximum_limit is not None and limit > self.maximum_limit:
```

Comment:
Redundant condition: `limit is not None`

Reply:
As for me, it’s not